lib/constants/api.js

```diff
@@ -1,3 +1,4 @@
 export const CONSORTIA_API = 'consortia';
 export const POLICIES_API = 'policies';
 export const ROLES_API = 'roles';
+export const ROLES_BY_USER_ID_API = 'roles/users';
```

Comment:
It's not only for getting roles users by ID, so maybe `ROLES_USERS_API` and `${ROLES_API}/users`?

Reply:
Agree, added your changes